packages/react-compass/src/accordion/accordion-button.tsx

```diff
@@ -33,6 +33,8 @@ const AccordionButton = React.forwardRef<HTMLButtonElement, ButtonProps>(
 
     return (
       <StyledAccordionButton
+        aria-expanded={expand === 'close' ? false : true}
```

Comment:
Use camel case please!

Reply:
React type definitions are case-sensitive for ARIA attributes